index.html

```diff
@@ -4239,23 +4237,19 @@ <h4><a name="11cICP">11.3.3.6 <span class="chunk">cICP</span>
 </tr>
 
 <tr>
-<td class="Regular">3</td>
+<td class="Regular">2</td>
 <td class="Regular">Matrix Coefficients</td>
 <td class="Regular">Describes the matrix coefficients used in deriving luma and chroma 
 signals from the green, blue and red as defined in 
 <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a><a</td>. 
-Currently, PNG is "RGB ONLY" and therefore a value of "1" is the default and indicates 
-"RGB Identity" for the "MATCOEFFS" chunk. Even though Matrix Coefficients aren't needed 
-for RGB sources, we are preserving the full CICP specification for future use-cases where 
-other color representations, that are typical in video imagery might be used 
-(i.e.- YCbCr, ICtCp).
+This value defaults to "0" or "RGB Identity", since PNG is explicitly RGB.
```

Comment:
I would say the value is always 0 for RGB data.
"Default" is imprecise. Does it mean if I specify some other value it gets treated as zero?

Reply:
Addressed this imprecise text between matrix coefficients and full-range flag (which is combined in H.273 and very imprecise as well).